Tests/Config/test_config.py

```diff
@@ -0,0 +1,33 @@
+import os
+import unittest
+from configparser import ConfigParser
+
+class TestConfig(unittest.TestCase):
+    def setUp(self):
+        self.path = os.path.join('Config/logger.ini')
+
+        if not os.path.isdir('Logs'):
+            os.mkdir('Logs')
+        if not os.path.exists(os.path.join('Logs', 'traceback.log')):
+            with open(os.path.join('Logs', 'traceback.log')) as fp:
```

Comment:
I have encountered an issue while running the test when traceback.log file isn't already created. Like in line 10, where Logs directory is created if not exists, we may need to create traceback.log if it doesn't exist.